companies/caterpillar.md

```diff
@@ -0,0 +1,25 @@
+---
+fortune_rank: 62
+name: Caterpillar
+tags: ["fortune500", "construction", "machinery", "tech", "insurance", "energy"]
+image: caterpillar.jpg
+blm_statements:
+  - url: https://www.caterpillar.com/en/company/the-fight-against-racism-and-injustice.html
+    date_posted: 2020-06-02
+    type: Blog
+    raw_text: Chairman and CEO Jim Umpleby shared the following message with employees,
```

Comment:
It looks like it's still erroring :/ Try this and make sure the entire block of raw_text is indented once more :)

```suggestion
    raw_text: >
       Chairman and CEO Jim Umpleby shared the following message with employees,
```